x-pack/plugins/observability_ai_assistant/public/rule_connector/obsaiassistant_connectors.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import type { ActionConnectorFieldsProps } from '@kbn/triggers-actions-ui-plugin/public';
+import { EuiFlexGroup, EuiFlexItem, EuiSelect } from '@elastic/eui';
+
+const ObsAIAssistantActionFields: React.FunctionComponent<ActionConnectorFieldsProps> = ({}) => {
+  return (
+    <EuiFlexGroup alignItems="center">
+      <EuiFlexItem grow={false}>
+        {i18n.translate('xpack.observabilityAiAssistant.alertConnector.selectLlmConnector', {
+          defaultMessage: 'Connector',
+        })}
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiSelect
+          data-test-subj="observabilityAiAssistantAlertConnectorSelect"
+          options={[{ value: 'azure-open-ai', text: 'azure-open-ai' }]}
+          value="azure-open-ai"
+        />
```

Comment:
These should be loaded with the [useGenAIConnectors](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_ai_assistant/public/hooks/use_genai_connectors.ts#L23) hook but this is a lazily loaded component with predefined properties and I don't think we can inject complex properties

Reply:
We expose a `useGenAIConnectors()` hook from the plugin's start contract that has service injected into it, so it doesn't need context - can we use that here as well?